### ISSUE TITLE
[BUG] Sidebar Search generates invalid href url. 

### ISSUE THREAD

**Issue by Jagdish-J-P** [jeroennoten/Laravel-AdminLTE#900]
### Describe the bug
There is a bug in sidebar search. I tried to check in issues but could not found if it is already reported or not.
When you search any navigation item in sidebar search, it is generating wrong URL.

### Steps To Reproduce
1. Search 'Password' in sidebar search.
2. Click on 'Change Password' from generated list.
3. It will open URL with 'http://127.0.0.1:8000/http%3A/127.0.0.1%3A8000/admin/settings'.
4. See error 404.

### Expected behavior
This link should be generated like http://127.0.0.1:8000/admin/settings.

### Screenshots
![bug screenshot](https://user-images.githubusercontent.com/20887370/119219789-e1f31680-bb04-11eb-8b02-21f6742b1ee3.png)

### Environment
Complete the next environment information.

| Item    | Version
| ------- | -------
| Laravel | 8.42.1
| Project | 3.6.0 
| OS      | Windows 10

**Comment by Shidersz** [comment]
@Jagdish-J-P This was a know bug on the underlying **AdminLTE v3.1** package. They already submit some fixes for it. Hopefully we see it fixed on the next release.

**Comment by diogocamarasbc** [comment]
Para contornar temporariamente a situação:

1-) **faça um backup** e abra o arquivo /public/vendor/adminlte/dist/js/adminlte.min.js
2-) procure por **escape(a.link)** -> provavelmente estará na linha 6, coluna 32987, posição 33174
3-) substitua escape(a.link) por apenas **a.link**
4-) salve e dê reload na página (ctrl+f5)
5-) teste.

EN >>

Temp workaround for this miserable situation:

1-) **backup** and open file /public/vendor/adminlte/dist/js/adminlte.min.js
2-) search for **escape(a.link)** -> it will probably be in line 6, column 32987, position 33174
3-) replace escape(a.link) by just **a.link**
4-) save and reload on browser (ctrl+f5)
5-) test it.

**Comment by benjaminv** [comment]
> Para contornar temporariamente a situação:
> 
> 1-) **faça um backup** e abra o arquivo /public/vendor/adminlte/dist/js/adminlte.min.js
> 2-) procure por **escape(a.link)** -> provavelmente estará na linha 6, coluna 32987, posição 33174
> 3-) substitua escape(a.link) por apenas **a.link**
> 4-) salve e dê reload na página (ctrl+f5)
> 5-) teste.
> 
> EN >>
> 
> Temp workaround for this miserable situation:
> 
> 1-) **backup** and open file /public/vendor/adminlte/dist/js/adminlte.min.js
> 2-) search for **escape(a.link)** -> it will probably be in line 6, column 32987, position 33174
> 3-) replace escape(a.link) by just **a.link**
> 4-) save and reload on browser (ctrl+f5)
> 5-) test it.

It is so interesting the minification can bring in errors. Thanks for the walkaround till the official fix.

**Comment by joe-er** [comment]
Looks like this has now been fixed but in 3.2.* of AdminLTE. Is it likely to get pulled through to here any time soon?

https://github.com/ColorlibHQ/AdminLTE/pull/3557

**Comment by joe-er** [comment]
@Shidersz ah very true - I missed that. Looking forward to them getting it sorted, can't wait :-)

**Comment by Shidersz** [comment]
@joe-er and others, I can confirm this is solved on the **pre-release** of **AdminLTE v3.2.0**. We going to wait to the release version, since there are some issues present on the **pre-release**.

**Comment by kyoheiS** [comment]
I can confirm that as of January 2022, we're still experiencing the same bug.
If anyone else is having trouble with this bug, please try the method below!
```
file: public\vender\adminlte\dist\js\adminlte.min.js
escape(a.link)
↓
a.link
```
